control-plane/stor-port/src/types/mod.rs

```diff
@@ -134,6 +134,10 @@ impl From<ReplyError> for RestError<RestJsonError> {
                 let error = RestJsonError::new(details, message, Kind::FailedPrecondition);
                 (StatusCode::PRECONDITION_FAILED, error)
             }
+            ReplyErrorKind::CapacityLimitExceeded => {
+                let error = RestJsonError::new(details, message, Kind::ResourceExhausted);
```

Comment:
Should we use kind CapacityLimitExceeded ?

Reply:
fixed